jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -91,6 +91,8 @@
         <configurationType implementation="software.aws.toolkits.jetbrains.services.lambda.execution.LambdaRunConfiguration"/>
         <runConfigurationProducer implementation="software.aws.toolkits.jetbrains.services.lambda.execution.sam.LambdaSamRunConfigurationProducer"/>
         <runConfigurationProducer implementation="software.aws.toolkits.jetbrains.services.lambda.execution.remote.LambdaRemoteRunConfigurationProducer"/>
+
+        <moduleBuilder builderClass="software.aws.toolkits.jetbrains.ui.wizard.java.SamInitModuleBuilder" />
```

Comment:
should this just be in the java extension?

Reply:
yes, but pycharm doesn't seem to try to call it. will move for safety